### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by daliannyvieira**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -428,6 +428,10 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
   - [InspirAda na Computação](https://inspiradanacomputacao.com/) - *Loc:* <img src="https://upload.wikimedia.org/wikipedia/en/0/05/Flag_of_Brazil.svg" alt="Brazil" width="30"> - *Lang: PT-BR only*
   
     Collaborative blog to give voice to women in computing, science and technology. The blog aims to share content in order to inspire women to become interested in the tech area. It promotes the access to the tech knowledge and brings, as a result, the equality of opportunities.
+    
+  - [OxenTIMenina](https://www.facebook.com/OxenTIMenina/) 
+  
+    Initiative that promotes technology, education and inspire girls in TI.
```
Comment (harsh183) [inline README.md]: 1. 'that' should be thats
2. What is TI, is it IT or something else, I don't think the readers will know over here.

In general this does not say much, can you elaborate, do they run workshops, help with internships, provide learning materials, run awareness programs?

**Comment by cristianoliveira** [comment]
Hey @daliannyvieira Thanks!

I searched a little about the project and found this website: http://oxentimenina.com.br/
Aren't the same as the facebook one? If so, could you please use it as the project link instead

How about this description adaptated from their's website?

"They aim to connect girls and women with the opportunity to have experience and make technology no matter which social class they're come from, their knowlodge or anything that is considered a limitation."

File: README.md (hunk 1 above)
Comment (daliannyvieira) [reply README.md]: Thanks for explaining :)

**Comment by daliannyvieira** [comment]
@cristianoliveira yes! is the same project! you can update! please

**Comment by harsh183** [comment]
I like @cristianoliveira 's description too. Can you merge yours and his and add it as a commit.

**Comment by cristianoliveira** [comment]
> @cristianoliveira yes! is the same project! you can update! please

Sure. Let me rebase here.

**Comment by cristianoliveira** [comment]
hey @daliannyvieira Thanks for your contributions! 

:)

**Pull request merged**